Bigtable/src/ResumableStream.php

```diff
@@ -175,11 +175,11 @@ public function readAll()
                         $currentAttempt = 0; // reset delay and attempt on successful read.
                     }
                 } catch (\Exception $ex) {
-                    $totalAttempt++;
-                    $currentAttempt++;
                 }
+                $totalAttempt++;
```

Comment:
please add a comment so we dont make the same mistake in the future :)

Reply:
also should there be a mutateRows specific test case to codify this behavior?

Reply:
> also should there be a mutateRows specific test case to codify this behavior?

The test below IS for `mutateRows` (it's called `testMutateRowsOnlyRetriesFailedEntries`), so I think we're covered, unless I'm missing something.